Assets/00_Content/Settings/PlayerInputActions.inputactions

```diff
@@ -43,7 +43,7 @@
                     "id": "0937b9b4-695c-41a9-b440-0d01345bde57",
                     "expectedControlType": "Button",
                     "processors": "",
-                    "interactions": ""
+                    "interactions": "Hold"
```

Comment:
This causes all uses of the interact action to requre you to hold the button. You should probably implement this in code instead so that others can use the action normally.